react-app/src/styles/Home.css

```diff
@@ -0,0 +1,36 @@
+@import url("https://fonts.googleapis.com/css2?family=Courier+Prime&display=swap");
+
+.home1 {
```

Comment:
The page is too "big". It has a scrollbar which stems from the fact that the home component has a height of 100vh and the navbar has its own height. One way to fix this would be to give the div tag in App.js a height of 100vh. Then you would give the navbar a specific height: 100px for example. Finally, you would write give height a value of 100% in this file. 100% just makes it take up as much space as is available based on the height of its parent element (the div in App.js). That way the home component would have a height of 100vh - height of navbar.